docs/src/content/docs/ops/backup-and-restore/backup-practices.adoc

```diff
@@ -0,0 +1,177 @@
+
+For Kafka-backed logs, we recommend the following practices to ensure transactional data remains available long enough to be flushed to storage or captured by a backup process:
+
+* **Replicate the log** - use a replication factor of at least `3` to tolerate broker or zone failures
+* **Enforce quorum writes** - set `min.insync.replicas > 1` to prevent data loss during partial outages
+* **Review retention policies** - ensure `log.retention.bytes` and `log.retention.ms` are tuned to retain unindexed messages long enough for safe processing
+
+XTDB sets safe defaults for producer settings, but we recommend reviewing topic-level configuration as detailed in the link:/ops/config/log/kafka[Kafka documentation^].
+
+Managed services such as link:https://www.confluent.io/confluent-cloud/[**Confluent Cloud**] offer stronger guarantees around availability, durability, and observability — and are recommended for production deployments with stricter RPO/RTO requirements.
+
+=== Why Back Up the Log?
+
+Backing up the log is not strictly required for all XTDB deployments. 
+In many cases, a full recovery can be achieved using the storage module alone.
+
+This is because XTDB periodically flushes indexed transactions from the log into immutable storage files. 
+As a result, the storage module acts as a form of backup for the log — capturing the state of the system at known log offsets.
+
+Flush behavior is controlled by:
+
+* A threshold number of transactions.
+* A maximum time interval between flushes (this defaults to 4 hours)
+
+These settings define your effective *Recovery Point Objective (RPO)* — that is, how much recent data you could lose in the event of log failure.
+
+However, backing up the log provides additional benefits in environments with stricter recovery requirements:
+
+* **Recover transactions submitted after the last flush** — reduces data loss compared to storage-only restores
+* **Avoid resetting the `epoch`** — restoring the log preserves continuity, allowing nodes to recover without configuration changes
+* **Faster point-in-time recovery** — restoring log and storage together may reduce bootstrap time and operational complexity
+
+=== Strategies for Protecting the Log
+
+There are several approaches to safeguarding the contents of the log. 
+These strategies may be used independently or in combination, depending on your system's tolerance for data loss, latency, and operational complexity.
+
+==== Point-in-Time Backups
+
+To capture a consistent snapshot of the log:
+
+* Backup the log *only after* a successful storage flush to avoid mismatches
+* Include only finalized segments or committed messages in the snapshot
+* Use infrastructure tooling (e.g. Kafka topic snapshotting or file-level copies for local logs) to extract consistent state
+
+[WARNING]
+====
+Always back up the *storage module first*, then the log.  
+Backing up the log before its associated storage state can result in unrecoverable mismatches during restoration.
```

Comment:
We can probably soften/clarify/delete the 'unrecoverable' here specifically, along the lines of: You would effectively only be recovering the storage (with a worse RPO) as the recovered log would be stale and cause a mismatch (resulting in the need for a new epoch)

I think it's also worth adding here that the delay for the log backup mustn't be more than the retention period of the log, or you'll end up in a similar unrecoverable position.